mchf-eclipse/drivers/audio/freedv_mchf.c

```diff
@@ -199,9 +203,11 @@ void fdv_print_ber()
 }
 
 void fdv_clear_display()
-{
+{ char clr_string[freedv_rx_buffer_max];
+
+  snprintf(clr_string,freedv_rx_buffer_max,"                                                                           ");
```

Comment:
Hi, of course something like memset would be much more efficient and straight forward..
The reason why it is still like that is, that I'm working on the intermediate font spacing, txt position and length, and am using the line of spaces for this work. In that case I'm copying the characters I'm currently working on to that line and can directly see the effect rather than working with received text...